tests/modules/home-environment/session-variables-non-nixos-expected.txt

```diff
@@ -0,0 +1,10 @@
+# Only source this once.
+if [ -n "$__HM_SESS_VARS_SOURCED" ]; then return; fi
+export __HM_SESS_VARS_SOURCED=1
+
+source "@nix@/etc/profile.d/nix.sh"
+
+
+export V1="v1"
+export V2="v2-v1"
+export XDG_DATA_DIRS="/nix/var/nix/profiles/default/share:$HOME/.nix-profile/share${XDG_DATA_DIRS:+:}$XDG_DATA_DIRS"
```

Comment:
CI fails due to my suggestion to use `config.home.profileDirectory`. We could just revert that (`$HOME/.nix-profile` doesn't really matter), or do something like following:

```suggestion
export XDG_DATA_DIRS="/nix/var/nix/profiles/default/share:@home@/.nix-profile/share${XDG_DATA_DIRS:+:}$XDG_DATA_DIRS"
```

And then below, in `tests/modules/home-environment/session-variables-non-nixos.nix`:

```nix
home = config.home.homeDirectory;
```